script/brave_license_helper.py

```diff
@@ -31,10 +32,120 @@ def AddBraveCredits(root, prune_paths, special_cases, prune_dirs,
         os.path.join('brave', 'vendor', 'brave-ios'),
         os.path.join('brave', 'vendor', 'brave_base'),
 
-        # These have auto-generated license files and
-        # GetThirdPartyDepsFromGNDepsOutput causes strange license errors
-        # unless the this entire directory is excluded.
-        os.path.join('brave', 'third_party', 'rust'),
+        # Metadata files for Rust crates are located in the subfolders of
+        # brave/third_party/rust/<crate_name>/<v>, the crates themselves in
+        # brave/third_party/rust/chromium_crates_io can be skipped.
+        os.path.join('brave', 'third_party', 'rust', 'chromium_crates_io'),
+
+        # Same for upstream
+        os.path.join('third_party', 'rust', 'chromium_crates_io'),
+
+        # Rust code written by Brave.
+        os.path.join('brave', 'third_party', 'rust', 'adblock_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'brave_news_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'brave_wallet'),
+        os.path.join('brave', 'third_party', 'rust',
+                     'challenge_bypass_ristretto_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'constellation_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'json_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'filecoin_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'skus'),
+        os.path.join('brave', 'third_party', 'rust', 'skus_cxx'),
+        os.path.join('brave', 'third_party', 'rust', 'speedreader'),
+        os.path.join('brave', 'third_party', 'rust', 'speedreader_ffi'),
+
+        # Rust crates that are dependencies of upstream rust and aren't actually
```

Comment:
We don't use any of these in Brave then?

Reply:
We may use some of them, but we don't have the crates under `//brave/third_party/rust`, they either in `//third_party/rust` and then we should get the credits for them when we process those, or the crates are not downloaded at all (I guess they are listed as dependencies, but the code from the isn't used?).